packages/salesforcedx-vscode-soql/src/editor/soqlEditorInstance.ts

```diff
@@ -194,6 +194,15 @@ export class SOQLEditorInstance {
   }
 
   protected handleRunQuery(): Promise<void> {
+    // Check to see if a default org is set.
```

Comment:
We decided it would be good to check for the default username before running a query as well. Without this, if there was no default org set and the user clicks "Run Query" then nothing would happen and there would be no feedback to the user. The messaging was hidden in the output channel.
Especially since we always allow our users to at least run a query regardless if it has unsupported syntax, we wanted to provide feedback as to why nothing was happening.